doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -211,28 +223,42 @@ Attributes of the ``uos_launcher`` specify the number of User VMs that the curre
   Specify the User VM memory size in Mbyte.
 
 ``gvt_args``:
-  GVT argument for the VM.
+  GVT argument for the VM. Recommendation is: 64 448 8. Leave it blank to disable the GVT.
```

Comment:
Do we need to enclose these numbers in quotes? (The `-G` parameter from `acrn-dm` takes it as `"64 448 8"` if I'm not mistaking).

I would also mention here that these arguments mean: `low_gm_size, high_gm_size, fence_sz`

```suggestion
  GVT arguments for the VM. Recommendation is: 64 448 8. Leave it blank to disable the GVT.
```

Reply:
the tool will help to add quotes automatically.